l5-operator/bundle.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM scratch
+
+# Core bundle labels.
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=l5-operator
+LABEL operators.operatorframework.io.bundle.channels.v1=alpha
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.15.0+git
```

Comment:
not sure if this could be an issue / inconsistency..

Reply:
Good call so I did a little test by creating a new operator and the result is the same. I will ask the Operator-sdk team about this.